app.rb

```diff
@@ -86,13 +88,63 @@ def rentals_of_person(id)
     end
   end
 
+  def save_data()
+    File.write('books.json', JSON.pretty_generate(@books))
+    File.write('people.json', JSON.pretty_generate(@people))
+    File.write('rentals.json', JSON.pretty_generate(@rentals))
```

Comment:
- [ ] Well! It appears that your `books.json` file is correctly saving the data. However, there seems to be an issue with the `people.json` and `rentals.json` files, as they are not saving the data correctly. As you can see in the screenshot below, the `name` values are being saved in the `age` field, and the `age` values are being saved in the `name` field. Please update your code to ensure that data is correctly saved in these files. Thank you for your attention to detail, and keep up the good work! 👍📚🔍
![image](https://github.com/Abdulwhbhussain/OOP-school-library/assets/55840999/35bbe263-dfdd-47b9-a98a-ff3d18944013)
- [ ] I also noticed that if I don't add a new people or rental and run the app again and exit, your files change and add proper data properly. But they should be done correctly in the first place without running the app again. Additionally, if I add a new people, some people have the right data, while others have wrong data. Please consider updating your code to address these issues.
![image](https://github.com/Abdulwhbhussain/OOP-school-library/assets/55840999/f023be38-58fc-4987-a399-b5ff5e6eec88)

Reply:
Fixed This Bug!